chemprop/v2/tests/unit/featurizers/test_molgraph.py

```diff
@@ -136,28 +136,28 @@ def test_invalid_bond_extra_shape(mol_featurizer, mol):
 
 
 def test_atom_extra_shape(mol, extra, atom_features_extra):
-    mf = MoleculeMolGraphFeaturizer(None, None, extra)
+    mf = MoleculeMolGraphFeaturizer(extra_atom_fdim=extra)
     mg = mf(mol, atom_features_extra=atom_features_extra)
 
     assert mg.V.shape == (mol.GetNumAtoms(), mf.atom_fdim)
 
 
 def test_atom_extra_values(mol, extra, atom_features_extra):
-    mf = MoleculeMolGraphFeaturizer(None, None, extra)
+    mf = MoleculeMolGraphFeaturizer(extra_atom_fdim=extra)
     mg = mf(mol, atom_features_extra=atom_features_extra)
 
     np.testing.assert_array_equal(mg.V[:, len(mf.atom_featurizer):], atom_features_extra)
 
 
 def test_bond_extra(mol, extra, bond_features_extra):
-    mf = MoleculeMolGraphFeaturizer(None, None, 0, extra)
+    mf = MoleculeMolGraphFeaturizer(extra_bond_fdim=extra)
     mg = mf(mol, bond_features_extra=bond_features_extra)
 
     assert mg.E.shape == (2 * mol.GetNumBonds(), mf.bond_fdim)
 
 
 def test_atom_bond_extra(mol, extra, atom_features_extra, bond_features_extra):
-    mf = MoleculeMolGraphFeaturizer(None, None, extra, extra, False)
+    mf = MoleculeMolGraphFeaturizer(extra_atom_fdim=extra, extra_bond_fdim=extra, bond_messages = True)
```

Comment:
`bond_messages = True` to `bond_messages=True`

Reply:
Changed.